documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -121,6 +121,13 @@
                   },
                   "commit_latency_bands":{
                      "$map_key=upperBoundOfBand": 1
+                  },
+                  "busiest_read_tag":{
+                     "tag": "",
+                     "fractional_cost": 0.0,
```

Comment:
Is there a place to explain how to interpret these two fields "fractional_cost" and "estimated_cost"?
I think it will be useful for DBA who operates this.

Reply:
All documentation for the tag throttling feature still needs to be written.